src/utils/Validation.js

```diff
@@ -0,0 +1,64 @@
+import HandleError from './Error.js';
+
+class Validation {
+  constructor() {
+    this.NAME_INPUT_ERROR = HandleError.NAME_INPUT_ERROR;
+    this.ATTEMPT_INPUT_ERROR = HandleError.ATTEMPT_INPUT_ERROR;
+  }
+
+  isValidNameLength(name) {
+    if (name.length > 5) {
+      HandleError.makeError(this.NAME_INPUT_ERROR.TooLongName);
+    }
+    return true;
+  }
```

Comment:
이름의 길이가 유효한지 확인할때 5라는 숫자는 항상 고정일까요?

만약 유효성을 확인해야 하는 숫자가 변경되면 새로운 함수를 만들어야 할 거 같아요.

확인해야하는 숫자를 인자로 받아 만든다면 조금 더 재사용할 수 있는 함수를 만들어볼 수 있을 거 같아요!
```suggestion
  isValidNameLength(name, length) {
    if (name.length > length) {
      HandleError.makeError(this.NAME_INPUT_ERROR.TooLongName);
    }
    return true;
  }
  
  // somewhere
  isValidNameLength(name, 5); 
```